### PR TITLE
SDK-257 : Add full_name and age attributes to PHP SDK

### PR DESCRIPTION
**Pull request by sidibos**

### CODE REVIEW

File: src/Yoti/ActivityDetails.php [hunk 1]
```diff
@@ -15,7 +15,9 @@ class ActivityDetails
 {
     const ATTR_FAMILY_NAME = 'family_name';
     const ATTR_GIVEN_NAMES = 'given_names';
+    const ATTR_FULL_NAME = 'full_name';
     const ATTR_DATE_OF_BIRTH = 'date_of_birth';
+    const ATTR_AGE_OVER = 'age_over:18';
```
Comment (vassyz) [inline src/Yoti/ActivityDetails.php]: This is not correct, this string is dynamic.

File: README.md [hunk 2]
```diff
@@ -115,8 +111,12 @@ $familyName         = $activityDetails->getFamilyName();
 
 $givenName          = $activityDetails->getGivenNames();
 
+$fullName           = $activityDetails->getFullName();
+
 $dateOfBirth        = $activityDetails->getDateOfBirth();
 
+$ageVerified        = $activityDetails->getProfileAttribute('age_over:18');
```
Comment (vassyz) [inline README.md]: Remove this.

File: example/profile.php [hunk 3]
```diff
@@ -50,6 +53,9 @@
                 <dt>Date Of Birth</dt>
                 <dd><?php echo $profile->getDateOfBirth() ?></dd>
 
+                <dt>Age verified</dt>
+                <dd><?php echo $profile->getProfileAttribute('age_over:18') ?></dd>
```
Comment (vassyz) [inline example/profile.php]: Can you assign this value somewhere at the top, and provide a comment, similar to the Ruby one:
```
This key uses the  format: age_[over|under]:[1-999] and is dynamically
generated based on the dashboard attribute Age / Verify Condition
```

Comment (vassyz) [reply example/profile.php]: Something like: `$ageVerified = $profile->getProfileAttribute('age_over:18')`

**Comment by vassyz** [comment]
Please also update the API Coverage. Example: https://github.com/getyoti/yoti-ruby-sdk/tree/SDK-254-new-attributes#api-coverage

File: README.md [hunk 4]
```diff
@@ -177,6 +178,23 @@ The example can be found in the [example folder](https://github.com/getyoti/php/
...
+
+* Activity Details
+    * [X] User ID `user_id`
+    * [X] Profile
+        * [X] Photo `selfie`
+        * [X] Given Names `given_names`
+        * [X] Family Name `family_name`
+        * [X] Full Name `full_name`
+        * [X] Mobile Number `phone_number`
+        * [X] Email address `email_address`
```
Comment (echarrod) [inline README.md]: a in **a**ddress is lowercase

**Pull request merged**